cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -4481,7 +4482,7 @@ instance Key KesKey where
 
     deterministicSigningKey :: AsType KesKey -> Crypto.Seed -> SigningKey KesKey
     deterministicSigningKey AsKesKey seed =
-        KesSigningKey (Crypto.genKeyKES seed)
+        KesSigningKey (Crypto.genKeyKES (error "TODO @node-team: fix this" seed))
```

Comment:
I think we'll need a call to [`sodiumInit`](https://github.com/input-output-hk/cardano-base/blob/8bd66f1109b48d9869f47c716ff51993631f7686/cardano-crypto-class/src/Cardano/Crypto/Libsodium/Init.hs#L14) somewhere.

Maybe in https://github.com/input-output-hk/cardano-node/blob/512a82974fb9b5745d1152a7ab0623f02e8fe45e/cardano-node/src/Cardano/Node/Handlers/TopLevel.hs#L65 or somewhere else.

Up to the node team (consensus is not responsible for propagating changes from `-base` to `-node` :grin:).